scripts/Globals/Globals.gml

```diff
@@ -1,31 +1,16 @@
 
-
 enum GameState {
 	Playing,
+	Crashing,
 	Finished
 }
 
-
 function initialise_globals() 
 {
-	
 	// DEBUG
 	global.show_debug = false
 
 	// SETUP
-	
 	global.display_width = 1920
 	global.display_height = 1080
-	
-	
-	// IN GAME
```

Comment:
nice, I wasn't sure how many objects would reference these so made them global when they didn't need to be